backend/api/Controllers/Models/EchoInspection.cs

```diff
@@ -43,18 +47,23 @@ public class EchoInspectionComparer : IEqualityComparer<EchoInspection>
         public bool Equals(EchoInspection? e1, EchoInspection? e2)
         {
             if (ReferenceEquals(e1, e2))
+            {
```

Comment:
All these extra lines will slow down the program immensely 🤢

Reply:
Fixed